### PR TITLE
Feature/73 changes

### PR DESCRIPTION
**Pull request by cookersjs**
Small PR that allows for email mid-UpdateDOIs and pushes all outputs to S3

### CODE REVIEW

**Review by SolomonShorser-OICR — DISMISSED** [review]

File: Jenkinsfile
Comment (jweiser) [inline Jenkinsfile]: Just curious why the space after the tilde was removed here?

**Review by jweiser — DISMISSED** [review]
Looks good!

File: Jenkinsfile
Comment (cookersjs) [reply Jenkinsfile]: That was a mistake, I've reverted it

**Comment by cookersjs** [comment]
@SolomonShorser-OICR @jweiser Had to do a small commit to fix something minor. Please approve again! (Also, is there any way to tune the sensitivity of what voids an approval?)

**Review by jweiser — APPROVED** [review]

**Comment by jweiser** [comment]
> @SolomonShorser-OICR @jweiser Had to do a small commit to fix something minor. Please approve again! (Also, is there any way to tune the sensitivity of what voids an approval?)

@cookersjs Re-approved!  Yes, the branch rules for te PRs has an option to dismiss an approval if there are more commits after approving.  At the time, I thought it would be a good idea to require re-approval after further commits, but we can disable it if it's causing too much of a headache having to approve and re-approve.

File: Jenkinsfile
Comment (SolomonShorser-OICR) [reply Jenkinsfile]: OMG really? it was a single space character... LOL ;)

**Comment by SolomonShorser-OICR** [comment]
You voided prior approval to add back a trivial space character? Ok, but next time I don't think you need to do that. ;)

**Review by SolomonShorser-OICR — APPROVED** [review]

**Comment by cookersjs** [comment]
I'm not entirely sure what the best idea is here, its not a huge deal to re-request approval, but it does make it a minor annoyance for small things like this. Not sure if that outweighs the benefits of requiring re-review for bigger fixes

**Pull request merged**
